internal/state/indexer/sink/psql/psql.go

```diff
@@ -177,12 +176,19 @@ INSERT INTO `+tableBlocks+` (height, chain_id, created_at)
 	})
 }
 
+var (
+	jsonpbMarshaller = jsonpb.Marshaler{
+		EnumsAsInts:  true,
+		EmitDefaults: true,
```

Comment:
Do we need this? Since we use proto3, the defaults are all zeroes, so it should be fine to skip them. (This option mainly exists for proto2, where there are nontrivial default values)